web/templates/settings/admin_profile_field_list.hbs

```diff
@@ -23,6 +23,14 @@
         </span>
         {{/if}}
     </td>
+    <td class="required_cell">
+        <span class="profile_field_required">
+            <label class="checkbox" for="profile_field_required_{{id}}">
+                <input class="required required_checkbox_{{required}}" type="checkbox" id="profile_field_required_{{id}}" {{#if required}} checked="checked" {{/if}} data-profile-field-id="{{id}}"/>
```

Comment:
`required` might be better called `required-field-toggle` to be more globally unique.

Also we've been generally using `-`s rather than `_`s` as delimiters in new code. See https://zulip.readthedocs.io/en/latest/subsystems/html-css.html#use-clear-unique-names-for-classes-and-object-ids.